app/src/main/res/layout/lessons_chapter_view.xml

```diff
@@ -21,7 +21,7 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:clickable="@{chapterSummary.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? true: false}"
-    android:minHeight="48dp"
+    android:minHeight="20dp"
```

Comment:
This is incorrect. We want this to be exactly same as before. 
As the issue describes you have to make change to `topic_lessons_story_summary` not this file. Also, we want the starting margin to be removed not the top/bottom.
In the above mentioned files you mainly need to make some change to `RecyclerView` to fix the issue.

Reply:
Actually I was not able to find the code where recycler view is initilized you must be talking about adding Item decorator to the recycler view so I was thinking to make another fresh PR. Please remove this one or should I do it from my end?

Reply:
> Actually I was not able to find the code where recycler view is initilized you must be talking about adding Item decorator to the recycler view so I was thinking to make another fresh PR. Please remove this one or should I do it from my end?

I am not talking about the item decorator, you just need to change the margin inside RecyclerView to make the left side margin(extra space) disappear. 

Also, you can close this PR, but clicking on `Close Pull Req`